doc/classes/ProjectSettings.xml

```diff
@@ -2752,11 +2752,17 @@
 			[/codeblock]
 		</member>
 		<member name="threading/worker_pool/low_priority_thread_ratio" type="float" setter="" getter="" default="0.3">
+			The ratio of threads to use as low-priority threads for multiprocessing tasks. The result is rounded down to the nearest integer. For example, the default ratio of [code]0.3[/code] on a system with 8 threads will result in 2 threads being used for low-priority tasks. Regardless of this value, the number of low-priority threads is always at least 1.
```

Comment:
What are some example tasks low-priority threads are used for in the engine?

Reply:
As of the time of me seeing this, I looked up both cases where `WorkerThreadPool::add_template_task` is used by the Engine and... none...!